scss/components/_cards.scss

```diff
@@ -271,3 +271,21 @@
     }
   }
 }
+
+.card--secondary {
+  .card__icon {
+    &.i-calendar {
+      @include u-icon-bg($calendar, $secondary-contrast);
```

Comment:
You deleted the calendar declaration above but are still using it here. That's why Travis is complaining https://travis-ci.org/18F/fec-style/builds/137429094#L1066